2022-03-08 01:02:00,447 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/.config/wandb/settings
2022-03-08 01:02:00,447 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/DP/Thesis/06_adapter/wandb/settings
2022-03-08 01:02:00,448 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-08 01:02:00,448 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-08 01:02:00,448 INFO    MainThread:3014 [wandb_init.py:_log_setup():386] Logging user logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_010200-3m8wo99a/logs/debug.log
2022-03-08 01:02:00,448 INFO    MainThread:3014 [wandb_init.py:_log_setup():387] Logging internal logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_010200-3m8wo99a/logs/debug-internal.log
2022-03-08 01:02:00,448 INFO    MainThread:3014 [wandb_init.py:init():420] calling init triggers
2022-03-08 01:02:00,448 INFO    MainThread:3014 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-08 01:02:00,449 INFO    MainThread:3014 [wandb_init.py:init():442] re-initializing run, found existing run on stack: 2r9t4juk
2022-03-08 01:02:00,451 INFO    MainThread:3014 [wandb_run.py:finish():1468] finishing run liebelife/adapter_CV_3x3/2r9t4juk
2022-03-08 01:02:00,451 INFO    MainThread:3014 [jupyter.py:save_history():429] not saving jupyter history
2022-03-08 01:02:00,452 INFO    MainThread:3014 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-08 01:02:00,452 INFO    MainThread:3014 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-03-08 01:02:00,452 INFO    MainThread:3014 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-08 01:02:00,456 INFO    MainThread:3014 [wandb_run.py:_restore():1769] restore
2022-03-08 01:02:01,088 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 01:02:01,203 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 01:02:01,431 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 01:02:01,573 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 9303
}

2022-03-08 01:02:01,678 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12763
}

2022-03-08 01:02:01,784 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12763
}

2022-03-08 01:02:01,887 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12763
  total_bytes: 12763
}

2022-03-08 01:02:01,991 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12763
  total_bytes: 12763
}

2022-03-08 01:02:02,093 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12763
  total_bytes: 12763
}

2022-03-08 01:02:02,196 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12763
  total_bytes: 12763
}

2022-03-08 01:02:02,298 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12763
  total_bytes: 12763
}

2022-03-08 01:02:02,980 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12763
  total_bytes: 12763
}

2022-03-08 01:02:03,424 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12763
  total_bytes: 12763
}
local_info {
}

2022-03-08 01:02:04,884 INFO    MainThread:3014 [wandb_run.py:_append_files():2194] logging synced files
2022-03-08 01:02:04,898 INFO    MainThread:3014 [wandb_init.py:init():471] starting backend
2022-03-08 01:02:04,899 INFO    MainThread:3014 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-08 01:02:04,902 INFO    MainThread:3014 [backend.py:ensure_launched():219] starting backend process...
2022-03-08 01:02:04,986 INFO    MainThread:3014 [backend.py:ensure_launched():225] started backend process with pid: 5450
2022-03-08 01:02:04,989 INFO    MainThread:3014 [wandb_init.py:init():480] backend started and connected
2022-03-08 01:02:05,049 INFO    MainThread:3014 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-08 01:02:05,050 INFO    MainThread:3014 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-08 01:02:05,050 INFO    MainThread:3014 [wandb_init.py:init():550] updated telemetry
2022-03-08 01:02:05,281 INFO    MainThread:3014 [wandb_init.py:init():581] communicating current version
2022-03-08 01:02:06,959 INFO    MainThread:3014 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-08 01:02:06,964 INFO    MainThread:3014 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-08 01:02:07,179 INFO    MainThread:3014 [wandb_init.py:init():624] starting run threads in backend
2022-03-08 01:02:12,191 INFO    MainThread:3014 [wandb_run.py:_console_start():1827] atexit reg
2022-03-08 01:02:12,192 INFO    MainThread:3014 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-08 01:02:12,192 INFO    MainThread:3014 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-08 01:02:12,194 INFO    MainThread:3014 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-08 01:02:12,194 INFO    MainThread:3014 [wandb_init.py:init():651] run started, returning control to user process
2022-03-08 01:02:32,542 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/.config/wandb/settings
2022-03-08 01:02:32,542 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/DP/Thesis/06_adapter/wandb/settings
2022-03-08 01:02:32,542 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-08 01:02:32,542 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-08 01:02:32,543 INFO    MainThread:3014 [wandb_init.py:_log_setup():386] Logging user logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_010232-1cdidnu0/logs/debug.log
2022-03-08 01:02:32,543 INFO    MainThread:3014 [wandb_init.py:_log_setup():387] Logging internal logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_010232-1cdidnu0/logs/debug-internal.log
2022-03-08 01:02:32,543 INFO    MainThread:3014 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fae1a1946d8>
2022-03-08 01:02:32,544 INFO    MainThread:3014 [wandb_init.py:init():420] calling init triggers
2022-03-08 01:02:32,544 INFO    MainThread:3014 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-08 01:02:32,544 INFO    MainThread:3014 [wandb_init.py:init():442] re-initializing run, found existing run on stack: 3m8wo99a
2022-03-08 01:02:32,547 INFO    MainThread:3014 [wandb_run.py:finish():1468] finishing run liebelife/adapter_CV_3x3/3m8wo99a
2022-03-08 01:02:32,547 INFO    MainThread:3014 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-08 01:02:32,552 INFO    MainThread:3014 [wandb_run.py:_restore():1769] restore
2022-03-08 01:02:33,253 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 01:02:33,448 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 01:02:33,651 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 01:02:34,262 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12764
}

2022-03-08 01:02:34,367 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12764
}

2022-03-08 01:02:34,475 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12764
}

2022-03-08 01:02:34,577 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}

2022-03-08 01:02:34,682 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}

2022-03-08 01:02:34,785 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}

2022-03-08 01:02:34,887 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}

2022-03-08 01:02:34,990 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}

2022-03-08 01:02:35,092 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}

2022-03-08 01:02:36,212 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}

2022-03-08 01:02:36,647 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12764
  total_bytes: 12764
}
local_info {
}

2022-03-08 01:02:38,111 INFO    MainThread:3014 [wandb_run.py:_append_files():2194] logging synced files
